src/einsteinpy/rays/shadow.py

```diff
@@ -35,15 +36,26 @@ def __init__(self, mass, n_rays, fov, limit=0.001):
         self.k0 = self._intensity()
         self.k1 = self._intensity_from_event_horizon()
         self.intensity = self.k1 + self.k0
-
+        def _compute_B(self):
+        '''
+        Returns an array of impact parameters
+        '''
+        return np.linspace(self.b_crit.value, self.distance, self.n_rays)
```

Comment:
was this function `_compute_B` required? I guess this is already handled by this.

https://github.com/einsteinpy/einsteinpy/blob/657330b0fda1f80c429f7eb025501d23fcdf848a/src/einsteinpy/rays/shadow.py#L25

Anyway, remove this change if you feel this was redundant.